testsuite/tests/formatting/test_locations.dlocations.ocamlc.reference

```diff
@@ -89,7 +89,7 @@ let rec fib = function | 0 | 1 -> 1 | n -> (fib (n - 1)) + (fib (n - 2))
       <def_rec>
         pattern (test_locations.ml[17,534+8]..test_locations.ml[17,534+11])
           Tpat_var "fib"
-          value_mode global,many,portable,unyielding;join(aliased,contended)(modevar#1[aliased,contended .. unique,uncontended])
+          value_mode global,many,portable,unyielding;imply(unique,uncontended)(modevar#1[aliased,contended .. unique,uncontended])
```

Comment:
Both sides have the same meaning. The change is probably due to specific implementation details and benign.

(note that internally, `aliased, contended` is in fact bottom, and `unique, contended` is top).